tapa/graphir_conversion/gen_rs_graphir.py

```diff
@@ -41,6 +41,7 @@
     get_task_graphir_parameters,
     get_task_graphir_ports,
     get_verilog_definition_from_tapa_module,
+    str_to_tokens,
```

Comment:
can we put this to graph ir as a static method, e.g. Expression.from_expr_str(str) -> Expression